building/tracks/syllabus/README.md

```diff
@@ -0,0 +1,126 @@
+# Syllabus
```

Comment:
Maybe it would be best to create a follow up issue about adding this somewhere with that link/quote, then we can think about it a bit more separately of this PR.